### DISCUSSION TITLE
More options for 'Value' field in Insights' Panels

### DISCUSSION THREAD

**Discussion by arladmin** [directus/directus#8687] [Feature Requests]
While configuring a 'Time Series' panel, in Insights: 
- for the 'Value' field, it would be nice to also have an option for 'Count of Records' along with the selected Collection's fields. 
Use-case: On the y-axis, display just the 'Count' of records for the corresponding (grouped or not) x-value instead of 'Field(count)'

**Comment by rijkvanzanten** [comment]
I'm probably misunderstanding exactly what you mean, but isn't that what the regular count aggregation already does?

<img width="319" alt="CleanShot 2021-10-12 at 13 28 42@2x" src="https://user-images.githubusercontent.com/9141017/137002305-05aa1026-6730-4c23-9ae8-f2bf832875c6.png">

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
